agones/Cargo.toml

```
@@ -24,8 +24,8 @@ description = "End to end integration tests to be run against a Kubernetes clust
 readme = "README.md"
 
 [dependencies]
-base64 = "0.13.0"
-k8s-openapi = { version = "0.16.0", features = ["v1_22"] }
-kube = { version = "0.76.0", features = ["runtime", "derive"] }
+base64.workspace = true
+k8s-openapi.workspace = true
+kube.workspace = true
```

I realised this morning - this may well break our agones e2e integration tests.

Reason being, `"rustls-tls"` doesn't work to authenticate with local `~/.kube` configuration and connect to an IP (https://github.com/kube-rs/kube/issues/153), although it works fine in cluster.

Something like:
```suggestion
kube.workspace = { workspace = true, features = ["runtime", "derive"] }
```

Will probably need to be utilised.